### PR TITLE
added a linked list implemented in python3

### PR DESCRIPTION
**Pull request by sapake**
for [issue #35](https://github.com/hacktoberfest17/programming/issues/35)

### CODE REVIEW

File: linked_list/linked_list.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    test_list.insert("I'm now first")
+    test_list.traverse()
+
+    print("\nDelete the node we first created:")
+    print("List size before delete: {}".format(test_list.size()))
+    test_list.delete("I'm first!")
+    test_list.traverse()
+    print("List size after delete: {}".format(test_list.size()))
+
+
```
Comment (t2013anurag) [inline linked_list/linked_list.py]: remove extra line

File: linked_list/linked_list.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
+#! /usr/bin/env python3
+
```
Comment (t2013anurag) [inline linked_list/linked_list.py]: remove extra line

File: linked_list/linked_list.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
+#! /usr/bin/env python3
```
Comment (t2013anurag) [inline linked_list/linked_list.py]: can you add a Readme file inside the same directory, explaining how the code works. Or links from where other contributors can learn.

Comment (sapake) [reply linked_list/linked_list.py]: Sure, I'll add a readme with explanation and some helpful links.

**Pull request merged**